File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int k=0;
        if(nums.size()==1)return 1;
        for(int i=0;i<nums.size();i++)
        {
            if(i==nums.size()-1||nums[i]!=nums[i+1])
            {
                k++;   
            }
            else{
                nums[i]=101;
            }
        }
        sort(nums.begin(),nums.end());
        return k;
    }
    
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
       int rd=0;
        for(int i=1;i<nums.length;i++)
        {
            if(nums[rd]!=nums[i])
            {
                rd++;
                nums[rd]=nums[i];
            }
        }
        return rd+1;
    }
}